src/components/Project/index.tsx

```diff
@@ -1,172 +0,0 @@
-import {
```

Comment:
Removing this file since projects aren't being rendered anymore on the home page. This source file can be added back to the source code once projects start being rendered again in the future.

<img width="809" alt="Screen Shot 2022-09-26 at 14 51 34" src="https://user-images.githubusercontent.com/48022589/192346177-8c9a995a-9f96-41a6-9f2e-4a75951ce7ff.png">